### PR TITLE
Fix error 

### PR DESCRIPTION
**Pull request by michael-mich**

### CODE REVIEW

File: src/controllers/game.ts [hunk 1]
```diff
@@ -3,6 +3,8 @@ import { ECharacterState, EShowOptions, EUserActions, EUserRace } from '../enums
 import { handleAttackEnemy, show as showFn } from './responses';
 import type { IAvailableCommands, IFightEntity, IFightTeam, IUserProfile } from '../types';
 import { createFight, getActiveFight, initProfile, leaveFight, saveLog, sendMessage } from '../communication';
+import type { AxiosResponse } from 'axios';
+import type { IDefaultResponse } from '../types';
```
Comment (RandomUsername12222) [inline src/controllers/game.ts]: Try to integrate this import with already existing imports. On line 5, all elements are also imported from types

File: src/controllers/game.ts [hunk 2]
```diff
@@ -244,7 +247,9 @@ export const newUserCommand = async (
         await prepareAdd(add, '', 'Incorrect race');
         renderAvailableRaces(add);
       } else {
-        await chooseRace(prepared[2], add);
+        const data = await chooseRace(prepared[2], add);
+        addProfile({ ...profile, ...data.data.state })
+        console.log(data.data.state)
```
Comment (RandomUsername12222) [inline src/controllers/game.ts]: Try to clean your code. Remove all unnecessary logs

File: src/controllers/game.ts (hunk 1 above)
Comment (michael-mich) [reply src/controllers/game.ts]: Integrate import on line 7 to imports on 4 line

File: src/controllers/game.ts (hunk 2 above)
Comment (michael-mich) [reply src/controllers/game.ts]: Remove console log

**Review by RandomUsername12222 — APPROVED** [review]

**Pull request merged**